### PR TITLE
feat : added navbar and folder structure

### PR DESCRIPTION
**Pull request by shaikhhamza11**
Need review on my folder structure.

### CODE REVIEW

File: src/components/Navbar/Navbar.css [hunk 1]
```diff
@@ -0,0 +1,115 @@
+#header {
```
Comment (SukanyaSen530) [inline src/components/Navbar/Navbar.css:1]: Suggestion - Try to avoid usage of ids.

**Review by SukanyaSen530 — APPROVED** [review]

File: src/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            </NavLink>{" "}
+          </li>
+          <i className="fa-solid fa-xmark fa-1x" id="close-navbar"></i>
+        </ul>
+      </nav>
+      <div id="hamburger">
+        <i
+          className="fa-solid fa-bars"
+          id="bar"
+          onClick={() => console.log("hello")}
```
Comment (spraveenofficial) [inline src/components/Navbar/Navbar.jsx:80]: You can remove console.log before pushing.

File: src/App.js [hunk 3]
```diff
@@ -1,41 +1,10 @@
 import "./App.css";
-import logo from "./logo.png";
-
+import Navbar from "./components/Navbar/Navbar";
+import Home from "./pages/Home/Home";
 function App() {
   return (
     <div className="App">
```
Comment (spraveenofficial) [inline src/App.js:6]: You can remove this default App Div.

**Review by spraveenofficial — COMMENTED** [review]
Suggested Changes.

**Pull request merged**